server/src/main/java/com/google/coffeehouse/servlets/LeaveClubServlet.java

```diff
@@ -0,0 +1,119 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.coffeehouse.servlets;
+
+import com.google.coffeehouse.storagehandler.StorageHandlerApi;
+import com.google.coffeehouse.common.Club;
+import com.google.coffeehouse.common.MembershipConstants;
+import com.google.coffeehouse.common.Person;
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.Map;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** 
+ * Servlet to leave a club from Http POST Request Body (in JSON format) containing a club ID
+ * and user ID that exists in the database, and return a success status 200 (OK).
```

Comment:
consider changing 'club ID', 'user ID' to 'clubId', 'userId' for consistency

Reply:
I figured because it should be kept as ID here because it's not talking about the name of the field passed in, but what it represents. Let me know if that makes sense though.